dotcom-rendering/src/web/components/HeaderTopBarSearchCapi.tsx

```diff
@@ -0,0 +1,211 @@
+import { css } from '@emotion/react';
+import {
+	brand,
+	brandAlt,
+	from,
+	neutral,
+	textSans,
+} from '@guardian/source-foundations';
+import { useEffect, useRef, useState } from 'react';
+import SearchIcon from '../../static/icons/search.svg';
+import { getZIndex } from '../lib/getZIndex';
+import { useApi } from '../lib/useApi';
+
+const searchLinkStyles = css`
+	${textSans.medium({ fontWeight: 'bold' })};
+	color: ${neutral[100]};
+	transition: color 80ms ease-out;
+	text-decoration: none;
+	padding: 7px 0;
+	background: none;
+	border: none;
+
+	${from.tablet} {
+		padding: 7px 10px 7px 5px;
+	}
+
+	:hover,
+	:focus {
+		text-decoration: underline;
+	}
+
+	svg {
+		fill: currentColor;
+		float: left;
+		height: 18px;
+		width: 18px;
+		margin: 3px 4px 0 0;
+	}
+	${getZIndex('searchHeaderLink')}
+`;
+
+const linkTablet = css`
+	display: none;
+	position: relative;
+
+	:before {
+		content: '';
+		border-left: 1px solid ${brand[600]};
+		height: 24px;
+	}
+
+	${from.desktop} {
+		display: flex;
+	}
+`;
+
+export const HeaderTopBarSearchCapi = () => {
+	const [isSearchVisible, setIsSearchVisible] = useState(false);
+	const [q, setQ] = useState<string>();
+	const qRef = useRef<HTMLInputElement>(null);
+	const { data } = useApi<{
+		response: { results: { webTitle: string; webUrl: string }[] };
+	}>(
+		q
+			? `https://content.guardianapis.com/search?api-key=test&q=${q}`
```

Comment:
Gotta love that api-key!

In all honesty, it would probably be best to create a dedicated one for this purpose, so usage can be tracked on the other side?

Reply:
Removed this - we probably shouldn't be leaking this - I'll have a think how we can get this in.

I don't want to use env vars for secrets as we can potentially deprecate that work now we don't use it for images.

Reply:
This will be present on the client-side… so there’s no benefit in obfuscating in here.